### DISCUSSION TITLE
Clean up interface: hide default Windows 10 window panels

### DISCUSSION THREAD

**Discussion by slaFFik** [spatie/ray#181] [Ideas]
Please check my screenshot:
![image](https://user-images.githubusercontent.com/282234/104811528-d31def80-5804-11eb-9120-9f1a867a3975.png)

I'm talking about these 2 lines: 
1. Ray and the minimize/maximize/close icons
2. Menu File Edit View Window Help

The top one can be integrated inside your app, similar to how Slack is doing:
![image](https://user-images.githubusercontent.com/282234/104811557-1b3d1200-5805-11eb-8bee-9ed591333ad6.png)

The menu items can be converted to a burger menu, again - similar to how Slack is doing:
![image](https://user-images.githubusercontent.com/282234/104811568-2ee87880-5805-11eb-8e0b-263ff96e8003.png)

These changes will clean up the interface SO MUCH.

**Comment by freekmurze** [comment]
@willemvb some nice suggestions here... 👍

**Comment by adevade** [comment]
I agree. The app looks so much cleaner in the macOS screenshots.
